Core/src/Seeding2/detail/CandidatesForMiddleSp2.cpp

```diff
@@ -221,8 +220,8 @@ void CandidatesForMiddleSp2::toSortedCandidates(
     }
 
     // Both have entries, get the minimum
-    if (descendingByQuality(spacePoints, m_storage[m_indicesLow[0]],
-                            m_storage[m_indicesHigh[0]])) {
+    if (m_storage[m_indicesLow[0]].weight >
+        m_storage[m_indicesHigh[0]].weight) {
       output[outIdx--] = m_storage[m_indicesHigh[0]];
```

Comment:
Not part of the diff but I'm not a huge fan of this use of operators. I think we previously discussed that this was copied as is.